masterlist.yaml

```diff
@@ -5218,20 +4897,14 @@ plugins:
         name: 'Beyond Skyrim: Bruma SE on Bethesda.net'
     group: *addonsGroup
     msg:
-      - <<: *hasPluginPatch
-        subs:
-          - 'Convenient Horses'
-          - '[Bruma patch for Convenient Horses](https://www.nexusmods.com/skyrimspecialedition/mods/13812/)'
+      - <<: *patchProvided
+        subs: [ 'Convenient Horses' ]
         condition: 'active("Convenient Horses.esp") and not active("CH Bruma Patch.esp")'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Flora Respawn Fix'
-          - '[Flora Respawn Fix SE for Beyond Skyrim](https://www.nexusmods.com/skyrimspecialedition/mods/11072/)'
+      - <<: *patchProvided
```

Comment:
The patch isn't provided at <https://www.nexusmods.com/skyrimspecialedition/mods/10917/>...

Reply:
I forgot about these thanks.

Reply:
Fixed in https://github.com/loot/skyrimse/pull/468/commits/bbf821e16285ff6a1764b402bea99665fedb6fa4